### ISSUE TITLE
EPIX doesn't make events except initial one after v0.1.2

### ISSUE THREAD

**Issue by mzks** [XENONnT/epix#37]
- Issue
EPIX returns only initial event, does not make instructions after that.

 - What I did to get it
This issue have observed in WFSim with Geant4 file simulation. In v0.1.1, this issue hasn't been observed.

- What I investigated to find the cause
With debugger, I found the cause comes from https://github.com/XENONnT/epix/blob/935ba6f4a4d24687bcaaf2721ccc43152ccd64b5/epix/run_epix.py#L115

After that, a several ten-s of `photons, electrons, excitons` has actual values, and the rest is 0.

The miminum example is here. (epix version v0.1.2)
```
$ ./run_epix --InputFile /dali/lgrandi/xenonnt/simulations/testing_epix_wfsim/tpc_and_nveto_cryoneutrons_200.root  --Debug --SourceRate 100
*** Detector definition message ***
You are currently using the default XENON10 template detector.

Using nestpy version 1.4.8
epix configuration:  {'input_file': '/dali/lgrandi/xenonnt/simulations/testing_epix_wfsim/tpc_and_nveto_cryoneutrons_200.root', 'detector': 'XENONnT', 'detector_config_override': '', 'cut_by_eventid': False, 'entry_start': None, 'entry_stop': None, 'micro_separation': 0.05, 'micro_separation_time': 10, 'tag_cluster_by': 'time', 'max_delay': 10000000.0, 'source_rate': 100.0, 'job_number': 0, 'output_path': '', 'debug': True, 'path': '/dali/lgrandi/xenonnt/simulations/testing_epix_wfsim', 'file_name': 'tpc_and_nveto_cryoneutrons_200.root', 'detector_config': [<epix.detector_volumes.SensitiveVolume object at 0x7ff4a65bea60>, <epix.detector_volumes.SensitiveVolume object at 0x7ff472db6fd0>], 'outer_cylinder': {'max_z': 7.3936, 'min_z': -154.6555, 'max_r': 66.4}}
Total entries in input file = 198
It took 1.7392 sec to load data.
Finding clusters of interactions with a dr = 0.05 mm and dt = 10 ns
It took 3.3007 sec to find clusters.
It took 3.2324 sec to merge clusters.
Removing clusters not in volumes: TPC BelowCathode
Number of clusters before: 2713
Number of clusters after: 2696
Assigning electric field to clusters
Generating photons and electrons for events
/home/mzks/official/epix/epix/quanta_generation.py:49: UserWarning: Energy deposition of 263.45294284820557 keV beyond NEST validity for NR model of 200 keV - Remove Interaction
  warnings.warn(f"Energy deposition of {en} keV beyond NEST validity for NR model of 200 keV - Remove Interaction")
/opt/XENONnT/anaconda/envs/XENONnT_development/lib/python3.8/site-packages/numpy/lib/function_base.py:2192: RuntimeWarning: invalid value encountered in ? (vectorized)
  outputs = ufunc(*inputs)
It took 0.8618 sec to get quanta.
Fixed event rate of 100.0 Hz
Done
Instructions saved to  tpc_and_nveto_cryoneutrons_200_wfsim_instructions.csv
It took 9.3694 sec to run epix.
```

```
$cat tpc_and_nveto_cryoneutrons_200_wfsim_instructions.csv
event_number,type,time,x,y,z,amp,recoil,e_dep,g4id,vol_id,local_field,n_excitons
0,1,64181824,19.575397,-62.51091,-38.670742,55,0,70.69686,11,1,200.0,54
0,1,64181824,21.687685,-58.404762,-38.932274,2834,8,58.00263,11,1,200.0,656
0,1,64181824,21.694613,-58.412357,-38.910706,3072,8,86.333595,11,1,200.0,897
0,1,64181824,21.70312,-58.441692,-38.907326,3908,8,94.53536,11,1,200.0,1113
0,1,64181824,21.708265,-58.3961,-38.94692,3390,8,81.34599,11,1,200.0,968
0,1,64181824,21.71108,-58.475006,-38.907944,6115,8,192.17854,11,1,200.0,2224
0,1,64181824,21.726461,-58.388557,-38.95812,3082,8,68.82258,11,1,200.0,798
0,1,64181824,21.746391,-58.384266,-38.96926,3888,8,98.20123,11,1,200.0,1151
0,1,64181824,21.76272,-58.39527,-38.949295,1648,7,29.4521,11,1,200.0,331
0,1,64181824,21.771029,-58.371216,-38.96195,1344,8,29.61329,11,1,200.0,309
0,1,64181824,21.771358,-58.37988,-38.969658,6643,7,159.28423,11,1,200.0,1745
0,1,64181824,21.782637,-58.37206,-38.96725,1249,8,28.531408,11,1,200.0,311
0,1,64181824,21.792103,-58.369766,-38.973232,1210,8,24.382124,11,1,200.0,255
0,1,64181824,21.802944,-58.372524,-38.975697,1870,8,35.91909,11,1,200.0,379
0,1,64181824,21.810009,-58.36582,-38.977016,1669,8,32.962933,11,1,200.0,404
0,1,64181824,21.816223,-58.35902,-38.979813,1437,8,31.141449,11,1,200.0,352
0,1,64181824,21.821163,-58.37088,-38.980762,1770,7,33.305866,11,1,200.0,388
0,1,64181824,21.82376,-58.356403,-38.982723,1343,8,26.706757,11,1,200.0,294
0,1,64181824,21.827198,-58.352818,-38.98878,870,8,17.632639,11,1,200.0,163
0,1,64181824,21.830738,-58.34713,-38.988533,1071,8,22.040703,11,1,200.0,221
0,1,64181824,21.835209,-58.3467,-38.98356,1148,8,21.878912,11,1,200.0,218
0,1,64181824,21.838873,-58.352108,-38.9808,1185,8,24.81744,11,1,200.0,247
0,1,64181824,21.842493,-58.356762,-38.979088,776,8,15.675897,11,1,200.0,135
0,1,64181824,21.846333,-58.365196,-38.967766,791,8,15.274351,11,1,200.0,130
0,1,64181824,21.846453,-58.377373,-38.95984,3765,8,83.938934,11,1,200.0,1007
0,1,64181824,21.847052,-58.358234,-38.975304,767,8,17.622437,11,1,200.0,170
0,1,64181824,21.848091,-58.36068,-38.97047,678,8,15.706513,11,1,200.0,140
0,1,64181824,21.855383,-58.36839,-38.94614,9332,8,323.62848,11,1,200.0,3613
0,1,64181824,27.006512,-48.68472,-39.216682,2404,7,46.001434,11,1,200.0,508
0,1,64181824,27.009596,-48.65983,-39.18694,6157,7,114.265724,11,1,200.0,1303
0,1,64181824,27.329262,-50.365635,-39.742687,101,7,2.6750844,11,1,200.0,4
0,1,64181824,27.72667,-49.465767,-39.161858,2285,7,42.793633,11,1,200.0,478
0,1,64181824,27.745577,-48.79939,-38.96093,4884,7,91.44903,11,1,200.0,1048
0,1,64181824,27.758389,-49.84528,-39.81129,8603,7,213.81401,11,1,200.0,2446
0,2,64181824,19.575397,-62.51091,-38.670742,8,0,70.69686,11,1,200.0,0
0,2,64181824,21.687685,-58.404762,-38.932274,1369,8,58.00263,11,1,200.0,0
0,2,64181824,21.694613,-58.412357,-38.910706,3280,8,86.333595,11,1,200.0,0
0,2,64181824,21.70312,-58.441692,-38.907326,3200,8,94.53536,11,1,200.0,0
0,2,64181824,21.708265,-58.3961,-38.94692,2588,8,81.34599,11,1,200.0,0
0,2,64181824,21.71108,-58.475006,-38.907944,8227,8,192.17854,11,1,200.0,0
0,2,64181824,21.726461,-58.388557,-38.95812,2005,8,68.82258,11,1,200.0,0
0,2,64181824,21.746391,-58.384266,-38.96926,3484,8,98.20123,11,1,200.0,0
0,2,64181824,21.76272,-58.39527,-38.949295,555,7,29.4521,11,1,200.0,0
0,2,64181824,21.771029,-58.371216,-38.96195,821,8,29.61329,11,1,200.0,0
0,2,64181824,21.771358,-58.37988,-38.969658,5156,7,159.28423,11,1,200.0,0
0,2,64181824,21.782637,-58.37206,-38.96725,776,8,28.531408,11,1,200.0,0
0,2,64181824,21.792103,-58.369766,-38.973232,486,8,24.382124,11,1,200.0,0
0,2,64181824,21.802944,-58.372524,-38.975697,752,8,35.91909,11,1,200.0,0
0,2,64181824,21.810009,-58.36582,-38.977016,785,8,32.962933,11,1,200.0,0
0,2,64181824,21.816223,-58.35902,-38.979813,868,8,31.141449,11,1,200.0,0
0,2,64181824,21.821163,-58.37088,-38.980762,661,7,33.305866,11,1,200.0,0
0,2,64181824,21.82376,-58.356403,-38.982723,679,8,26.706757,11,1,200.0,0
0,2,64181824,21.827198,-58.352818,-38.98878,398,8,17.632639,11,1,200.0,0
0,2,64181824,21.830738,-58.34713,-38.988533,504,8,22.040703,11,1,200.0,0
0,2,64181824,21.835209,-58.3467,-38.98356,483,8,21.878912,11,1,200.0,0
0,2,64181824,21.838873,-58.352108,-38.9808,620,8,24.81744,11,1,200.0,0
0,2,64181824,21.842493,-58.356762,-38.979088,341,8,15.675897,11,1,200.0,0
0,2,64181824,21.846333,-58.365196,-38.967766,329,8,15.274351,11,1,200.0,0
0,2,64181824,21.846453,-58.377373,-38.95984,2647,8,83.938934,11,1,200.0,0
0,2,64181824,21.847052,-58.358234,-38.975304,532,8,17.622437,11,1,200.0,0
0,2,64181824,21.848091,-58.36068,-38.97047,510,8,15.706513,11,1,200.0,0
0,2,64181824,21.855383,-58.36839,-38.94614,14530,8,323.62848,11,1,200.0,0
0,2,64181824,27.006512,-48.68472,-39.216682,1026,7,46.001434,11,1,200.0,0
0,2,64181824,27.009596,-48.65983,-39.18694,2269,7,114.265724,11,1,200.0,0
0,2,64181824,27.329262,-50.365635,-39.742687,101,7,2.6750844,11,1,200.0,0
0,2,64181824,27.72667,-49.465767,-39.161858,878,7,42.793633,11,1,200.0,0
0,2,64181824,27.745577,-48.79939,-38.96093,1831,7,91.44903,11,1,200.0,0
0,2,64181824,27.758389,-49.84528,-39.81129,7304,7,213.81401,11,1,200.0,0
```
No lines are omitted. It takes care about the only first event.

Conclusions / guess
I haven't go into the function but I guess it may come from nestpy or its versions. I couldn't find the cause.
Could someone provide an example which WFSim works well with the latest epix?

**Comment by mzks** [comment]
`nestpy` is not in the list of `requirements.txt` thus the version hasn't been specified.

**Comment by ramirezdiego** [comment]
Fixed in #34
